runtime/src/main/java/org/corfudb/runtime/view/stream/ReplexStreamView.java

```diff
@@ -49,6 +50,46 @@ public ReplexStreamView(final CorfuRuntime runtime,
 
     /** {@inheritDoc}
      *
+     * This is problematic in Replex since we need to translate
```

Comment:
We are unable to? The unit tests run fine and don't affect any other code. I don't think it's a good idea to hastily remove it at this point.